ReSharper.FSharp/test/data/features/daemon/lambdaAnalyzer/Delegates.fs

```diff
@@ -15,6 +15,8 @@ Delegate(fun x -> x)
 A.M1(fun x -> x)
 A.M1(0, fun x -> x)
 A.M2(0, fun x -> x)
+A.M2(0, y = fun x -> x)
```

Comment:
Is `Delegates.fs` the correct test file to add these function parameter cases to?

Reply:
this file contains cases that, in fact, check for false the ```isImplicitlyConvertedToDelegate``` predicate in the lambda analyzer